### PR TITLE
docs: Add Manual Testing CloudInit CRDs

### PR DESCRIPTION
**Pull request by irishgordo**
* manual testing docs for Harvester CloudInit CRDs

Resolves: feat/harvester-cloud-init
See also: https://github.com/harvester/harvester/issues/3902

### CODE REVIEW

File: docs/content/manual/misc/3902-elemental-cloud-init-harvester-crds.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+        - authorized_keys:
+            rancher:
+              - ATTACH-YOUR-SSH-KEY-HERE
+```
+2. Apply it to the cluster via `kubectl create -f filename.yaml`
+3. validate that the CloudInit resource was built appropriately `kubectl describe cloudinits/ssh-access-test-one-node`
+4. Reboot the node that has the host-name filed in, as in the node tied to `kubernetes.io/hostname`
+5. Validate that you can ssh as rancher onto the node that was rebooted without the need for password
+6. Delete the CRD `kubectl delete cloudinits/ssh-access-test-one-node`
+7. Remove the ssh-key from `/home/rancher/.ssh/authroized_keys`
```
Comment (albinsun) [inline docs/content/manual/misc/3902-elemental-cloud-init-harvester-crds.md:141]: typo, should be `authorized`.

File: docs/content/manual/misc/3902-elemental-cloud-init-harvester-crds.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+```yaml
+apiVersion: node.harvesterhci.io/v1beta1
+kind: CloudInit
+metadata:
+  name: silly-file-one-node-no-changes
+spec:
+  matchSelector:
+    kubernetes.io/hostname: harvester-node-0
+  paused: false
+  filename: 99_filewrite_singelnode_wont_get_changes.yaml
```
Comment (albinsun) [inline docs/content/manual/misc/3902-elemental-cloud-init-harvester-crds.md:186]: typo, should be `single`?

File: docs/content/manual/misc/3902-elemental-cloud-init-harvester-crds.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+```yaml
+apiVersion: node.harvesterhci.io/v1beta1
+kind: CloudInit
+metadata:
+  name: silly-file-one-node-no-changes
+spec:
+  matchSelector:
+    kubernetes.io/hostname: harvester-node-0
+  paused: true
+  filename: 99_filewrite_singelnode_wont_get_changes.yaml
```
Comment (albinsun) [inline docs/content/manual/misc/3902-elemental-cloud-init-harvester-crds.md:207]: typo, should be `single`?

**Review by albinsun — CHANGES_REQUESTED** [review]

File: docs/content/manual/misc/3902-elemental-cloud-init-harvester-crds.md [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+  filename: 99_filewrite.log
+  contents: |
+    stages:
+      fs:
+        - name: "write file test"
+          commands:
+            - echo "hello, there" > /etc/sillyfile.conf
+```
+2. Log on to any one of the nodes in the cluster and validate:
+  1. `99_filewrite.log.yaml` is present within `/oem/` directory on the node
```
[the remark this answers is not in the document]
Comment (irishgordo) [reply docs/content/manual/misc/3902-elemental-cloud-init-harvester-crds.md:46]: Thanks for this catch :smile:  - I've added another two spaces on that.
Rendering with `hugo server` validated that it does come across cleaner :smile: 
Screenshot from 2024-02-15 11-38-34

File: docs/content/manual/misc/3902-elemental-cloud-init-harvester-crds.md (hunk 3 above)
Comment (irishgordo) [reply docs/content/manual/misc/3902-elemental-cloud-init-harvester-crds.md]: Thanks for this catch :smile: :+1: - updated & force pushed fixing

File: docs/content/manual/misc/3902-elemental-cloud-init-harvester-crds.md (hunk 1 above)
Comment (irishgordo) [reply docs/content/manual/misc/3902-elemental-cloud-init-harvester-crds.md]: Thanks for this catch :smile: :+1: - updated & force pushed fixing

File: docs/content/manual/misc/3902-elemental-cloud-init-harvester-crds.md (hunk 2 above)
Comment (irishgordo) [reply docs/content/manual/misc/3902-elemental-cloud-init-harvester-crds.md]: Thanks for this catch :smile: :+1: - updated & force pushed fixing

**Review by albinsun — APPROVED** [review]
LGTM

**Pull request merged**